conformance/operator_test.go

```diff
@@ -701,6 +701,109 @@ var _ = Describe("operator", func() {
 					})))
 			})
 		})
+		Context("MTU", func() {
+			var mtuNetwork *sriovv1.SriovNetwork
+
+			BeforeEach(func() {
+				node := sriovInfos.Nodes[0]
+				intf, err := sriovInfos.FindOneSriovDevice(node)
+				Expect(err).ToNot(HaveOccurred())
+
+				mtuPolicy := &sriovv1.SriovNetworkNodePolicy{
```

Comment:
Once [PR22 ](https://github.com/openshift/sriov-tests/pull/22) is in, you can use the network utils:
[CreateSriovPolicy
](https://github.com/openshift/sriov-tests/pull/22/files#diff-990b2c01963c93af24003afbdb0b86b2R34) with some additional changes.
If this PR is in first, I'll update it once 22 is merged.